src/aiParams.ts

```diff
@@ -73,6 +67,7 @@ export interface CustomModel {
   isBuiltIn?: boolean;
   enableCors?: boolean;
   core?: boolean;
+  azureOpenAIApiDeploymentName?: string; // Added for Azure OpenAI models
```

Comment:
Isn't this the same as the one above?

Reply:
I'm just trying to remove the last 2 commits I made - struggling.

Reply:
But to answer your question, currently when choosing custom model, you are not given a choice of selecting a model with your deployment ID. I might be wrong, but my testing has made me believe that several gpt-4o models are available on one azure openai deployment.